sdk/tables/azure-data-tables/README.md

```diff
@@ -1,76 +1,70 @@
 # Azure Data Tables client library for Python
 
-Azure Data Tables is a NoSQL data storing service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
-Tables scale as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
+Azure Data Tables is a NoSQL data storage service that can be accessed from anywhere in the world via authenticated calls using HTTP or HTTPS.
+Tables scales as needed to support the amount of data inserted, and allow for the storing of data with non-complex accessing.
 The Azure Data Tables client can be used to access Azure Storage or Cosmos accounts.
 
-
-# Usage
-* Storing structured data in the form of tables	# Usage
-* Quickly querying data using a clustered index
-
-[Source code](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk) | [Package (PyPI)](https://pypi.org) | [API reference documentation](https://aka.ms/azsdk/python/tables/docs) | [Product documentation](https://docs.microsoft.com/azure/storage/) | [Samples](https://github.com/Azure/azure-sdk-for-python/tree/master/sdk)
-For code examples, see [MyService Management](https://docs.microsoft.com/python/api/overview/azure/)
-on docs.microsoft.com.
+[Source code][source_code] | [Package (PyPI)][Tables_pypi] | [API reference documentation][Tables_ref_docs] | [Samples][Tables_samples]
```

Comment:
tables ref docs should also be an aka link that points to the github docs, i.e. [these](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-identity/1.4.0/azure.identity.html#azure.identity.DefaultAzureCredential)